exporters/otlp/trace/build.gradle.kts

```diff
@@ -28,6 +28,10 @@ dependencies {
     implementation("io.grpc:grpc-stub")
     implementation("com.google.protobuf:protobuf-java")
 
+    implementation("com.squareup.okhttp3:okhttp:4.9.0")
```

Comment:
Ok! I'm inclined to keep the surface area smaller and choosing to either shade or not shade. Can always publish additional artifacts later, but it's harder to stop publishing.

Reply:
On second thought, maybe okhttp3 isn't such a great idea. Its only getting critical fixes [through December 31st, 2021](https://square.github.io/okhttp/#requirements). And since we're going with the separate module route, it doesn't seem as critical that the kotlin stdlib is included.

Reply:
3.12.x isn't getting updates, but the 4.x will be.

Reply:
Ah I didn't know about the EoL then seems ok (indeed especially with the separate module)